bip-0008.mediawiki

```diff
@@ -72,8 +72,9 @@ for the purposes of this proposal, and support two future upgrades for different
 When a block nVersion does not have top bits 001, it is treated as if all
 bits are 0 for the purposes of deployments.
 
-Miners should continue setting the bit in LOCKED_IN phase so uptake is visible, though this has no effect on
-consensus rules.
+Miners must continue setting the bit in LOCKED_IN phase so uptake is visible and acknowledged.
+Blocks without the applicable bit set are invalid during this period.
```

Comment:
No.  This requires instant changes upon LOCKED_IN, which was never the intent.  Miners with BIP9 didn't have to change anything for 2 weeks.